index.html

```diff
@@ -173,6 +173,16 @@ <h3 class="grouping-title">Attributes</h3>
               the built-in scene lights. Refer to <a href="https://wiki.jmonkeyengine.org/jme3/advanced/pbr_part3.html" target="_blank">this article</a>
               for in-depth details related to this lighting mode.</p>
             </li>
+            <li>
+              <div>interaction-prompt</div>
+              <p>Allows you to change the conditions under which the visual and
+              audible interaction prompt will display. Allowed values are "auto"
+              and "when-focused". If set to "auto", the interaction prompt will
+              be displayed as soon as the interaction-prompt-threshold (see below)
+              time has elapsed (after the model is revealed). If set to "when-focused",
```

Comment:
I don't think the prompt should come up until the model is visible on the page, hence my comment about modelIsVisible not meaning much. Try this with auto on our lighting examples page. You'll notice as you scroll that the prompt has no delay except on the first one.

Reply:
Yeah, acknowledged. I think we should work on this as a separate improvement when we square away what we mean by `modelIsVisible`.

Reply:
(as part of #524 , that is)